coremark/examples/coremark_example/pytest_coremark.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
```

Comment:
not related but now that you're updating the coremark, should we fix this as well? :stuck_out_tongue_closed_eyes:  https://github.com/espressif/idf-extra-components/issues/368

Reply:
Done in https://github.com/espressif/idf-extra-components/pull/383 :)